release/src/main/scripts/publish_docker_images.sh

```diff
@@ -39,25 +39,24 @@ RC_VERSION="rc${RC_NUM}"
 
 echo "================Confirming Release and RC version==========="
 echo "We are using ${RC_VERSION} to push docker images for ${RELEASE}."
+echo "Publishing the following images:"
+docker images --filter "reference=apache/beam_*:${RELEASE}_${RC_VERSION}" --format "{{.Repository}}" | while read IMAGE; do
+  echo "${IMAGE}"
+done
 echo "Do you want to proceed? [y|N]"
 read confirmation
 if [[ $confirmation = "y" ]]; then
-
-  IMAGES=$(docker images --filter "reference=apache/beam_*:${RELEASE}_${RC_VERSION}" --format "{{.Repository}}")
-  echo "Publishing the following images:" "${IMAGES[@]}"
-  echo "Make sure the images are correct, then press any key to proceed."
-  read
-  for IMAGE in "${IMAGES[@]}"; do
+  docker images --filter "reference=apache/beam_*:${RELEASE}_${RC_VERSION}" --format "{{.Repository}}" | while read IMAGE; do
```

Comment:
Can "reference=apache/beam_*:${RELEASE}_${RC_VERSION}" --format "{{.Repository}}" part be moved to a variable to maintain consistency ?

Reply:
done